website/docs/cloud-docs/agents/index.mdx

```diff
@@ -127,6 +127,16 @@ Agents are organized into pools, which can be managed in Terraform Cloud's UI. E
 
    ![Screenshot: Unable to delete an agent pool with attached workspace](/img/docs/agent-delete-invalid.png)
 
+### Scope an Agent Pool to specific workspaces
+
+1. Navigate to **Organization Settings > Agents** and click on the name of the pool you would like to update.
+
+1. Click "Grant access to specific workspaces" option. This will open a new search menu for searching workspaces within this organization.
+
+1. Start typing the name of the workspace you would like to grant access to the agent pool and select it from the drop down list.
+
+1. Click "Save".
```

Comment:
Can you select more than one workspace from the list at a time? 

```suggestion
1. Click **Grant access to specific workspaces**. A menu opens where you can search for workspaces within this organization.
1. Select the workspace you would like to grant access to the agent pool.
1. Click **Save**.
```
```suggestion
1. Click "Grant access to specific workspaces" option. This will open a new search menu for searching workspaces within this organization.

1. Start typing the name of the workspace you would like to grant access to the agent pool and select it from the drop down list.

1. Click "Save".
```